### DISCUSSION TITLE
Accessible build verbosity issue

### DISCUSSION THREAD

**Discussion by Bhrighde** [jamulussoftware/jamulus#2595] [Technical Support & Help]
We have a couple users of the accessible Jamulus build.  While we appreciate having a version that works with screen readers for blind and visually-impaired musicians, there is a major issue regarding extreme verbosity when navigating through the program.  Is it possible to make adjustments to this so we don’t have to listen to our screen reading software read every single bit of text on the screen before we hear where we are (for example, when tabbing from “mute” to “solo” on a channel)?

**Comment by hoffie** [comment]
Hi,

which build are you referring to exactly? I assume it's one by @chigkim (related: #490 and #1466)?
Sadly, I'm not familiar with accessibility details at all, for example, how a program is supposed to behave and how a (Qt) program can be made to behave that way.
We'd certainly be open to improvements in this area, but I'm not sure if someone is actively working on that part right now.

**Reply by Bhrighde** [reply]
Its #1466.

**Reply by ann0see** [reply]
@chigkim is providing the fork, so I think you need to get in touch with him (via his repo probably?)
